src/spring/HISTORY.md

```diff
@@ -1,13 +1,11 @@
 Release History
 ===============
 
-1.1.7
+1.1.8
```

Comment:
We'll skip `1.1.6` and `1.1.7` on purpose

Reply:
But we only hide version `1.1.6` at present, so the version `1.1.7` has not been released, right?

Reply:
Yes, 
1. `1.1.7` hasn't been updated into `setup.py`.
2. Also `1.1.7` hasn't been merged into `index.json`.